From: Joel Selvaraj <foss@joelselvaraj.com>
Date: Tue, 27 Aug 2024 20:36:06 -0500
Subject: [PATCH] sdm845.config: enable CS35L36 speaker amp codec

Used in Google Pixel 3.

Signed-off-by: Joel Selvaraj <foss@joelselvaraj.com>
(cherry picked from commit 9bd98192e0a272531364962b6e9f02a4ade1ab42)
Link: https://gitlab.com/sdm845-mainline/linux
Signed-off-by: Richard Acayan <mailingradian@gmail.com>
---
 arch/arm64/configs/sdm845.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/sdm845.config b/arch/arm64/configs/sdm845.config
index b770968..c59c637 100644
--- a/arch/arm64/configs/sdm845.config
+++ b/arch/arm64/configs/sdm845.config
@@ -32,6 +32,7 @@ CONFIG_SND_SOC_TFA989X=m
 # Pixel 3
 CONFIG_DRM_PANEL_LG_SW43408=y
 CONFIG_TOUCHSCREEN_STM_FTS_DOWNSTREAM=m
+CONFIG_SND_SOC_CS35L36=m
 
 # Odin
 # Driver has been dropped
